examples/ltc_backend/CMakeLists.txt

```diff
@@ -0,0 +1,94 @@
+if ("${TORCH_DIR}" STREQUAL "")
+    get_filename_component(TORCH_DIR "${CMAKE_CURRENT_SOURCE_DIR}/../../../pytorch/" ABSOLUTE)
+endif()
+message(STATUS "TORCH_DIR=${TORCH_DIR}")
+
+if ("${TORCH_MLIR_DIR}" STREQUAL "")
+    get_filename_component(TORCH_MLIR_DIR "${CMAKE_CURRENT_SOURCE_DIR}/../../" ABSOLUTE)
```

Comment:
can we use https://cmake.org/cmake/help/latest/variable/PROJECT_SOURCE_DIR.html ?